upi/openstack/02_network.yaml

```diff
@@ -156,7 +168,8 @@
     command:
       cmd: "openstack floating ip set --port {{ os_port_ingress }} {{ os_ingress_fip }}"
 
-  - name: 'Set MTU for the first neutron network'
+
+  - name: 'Create SNAT subnet'
     command:
-      cmd: "openstack networrk set {{ os_network }} --mtu {{ neutron_network_mtu }}"
+      cmd: "neutron subnet-create {{ os_external_network }} {{ aci_cni['snat_host_pool'][subnet] }} --name snat-subnet-pool --disable-dhcp --gateway {{ aci_cni['snat_host_pool']['gateway'] }} --apic:snat_host_pool True"
```

Comment:
as per hierarchy should be aci_cni['network_interfaces']['node']['snat_host_pool']['subnet'] and aci_cni['network_interfaces']['node']['snat_host_pool']['gateway'] right?

Reply:
Good catch, fixed!